.gitignore

```diff
@@ -1,2 +1,2 @@
 node_modules
-
+dist
```

Comment:
I was getting so many npe and undefined errors I decided I had to typescriptify to even be sure the thing worked. Not used to working in loosely typed languages anymore!